cnxarchive/views.py

```diff
@@ -557,6 +561,27 @@ def search(environ, start_response):
     status = '200 OK'
     headers = [('Content-type', 'application/json')]
     start_response(status, headers)
+    
+    #In the case where a search is performed with an authorId as a filter,
+    # it is possible for the database to return no results even if the author exists in the database.
+    #  Therefore, the database is queried a second time for contact information associated with only the authorIds.
+    #  The author information is then used to update the results returned by the first database query. 
+    if len(db_results)<=0 :
+        authors_list =[]
+        keyword_list = query.filters + query.terms + query.sorts
+        for key, value in keyword_list:
```

Comment:
`keyword_list` includes `query.sorts` which is just a single value.  So `keyword_list` looks like this for `cnxarchive.tests.test_views.ViewsTestCase.test_search_unbalanced_quotes`:

```
[('text', 'a phrase'), ('text', 'something else'), ('author', 'first last'), 'pubDate']
```

May be (I'm not sure) it's not necessary to include `query.sorts`?